data/piratebay

```diff
@@ -0,0 +1,11 @@
+# Piratebay forum
+pirates-forum.org
+
+# Piratebay image hosting site
+torrindex.net
```

Comment:
Does this image hosting site only serve pirate bay?

Reply:
I believe that `torrindex.net` only serves `thepiratebay.org`, but it might be not limited only to images, but it might contain other static contents, too. It is mentioned in [torrentfreak](https://torrentfreak.com/the-pirate-bays-main-domain-returns-after-month-of-downtime-200411/).

Reply:
Then you may move this domain into another category file.

Reply:
Since `torrindex.net` is exclusively used by `thepiratebay.org`, I think that these two websites should be under the same category. Also, my ISP has blocked these domains together to avoid access to `thepiratebay.org` and its mirrors.
What separate category would you suggest for `torrindex.net`?

Reply:
IDK, let's hear from other contributors.